.gitignore

```diff
@@ -279,3 +279,4 @@ build/
 install_script.sh
 .cache/
 release.sh
+MANIFEST
```

Comment:
The MANIFEST should be added to git otherwise when someone else clones the repo, they'd run into the issue you had with not finding HISTORY.rst on set up right?